### DISCUSSION TITLE
Error compiling the demo

### DISCUSSION THREAD

**Discussion by sangohan** [HaxeGodot/godot#11] [General]
I downloaded the latest version of the demo (https://github.com/HaxeGodot/squash-the-creeps-3d) which i tried to compile. I get an error: 

```
> haxe -version
4.2.1+bf9ff69

> haxe build.hxml
C:\apps\dev\haxe\haxe\lib\godot/3,4,1/godot/collections/Array.hx:52: characters 10-44 : Warning : No overload found for this constructor call
```

I used this version of haxe as it is what is used by the github actions (https://github.com/HaxeGodot/godot/blob/main/.github/workflows/main.yml#L18). The latest haxe release seems to also fail (but at a different place - likely an `import.hx` issue, but didn't debug or investigate).

**Comment by ibilon** [comment]
This is only a warning and not an error, the demo was still compiled successfully.

The godot code generate many of these  "No overload found for this constructor call" warnings, I couldn't figure out why, but they don't seem to be problematic, so you can ignore them.

**Reply by sangohan** [reply]
Ah that's interesting. The godot project fails to open with missing Main.cs file in the build directory, and i thought it was because the compilation was not working, but turns out it was because i wasn't using the mono version of godot. 

[PEBKAC ](https://en.wikipedia.org/wiki/User_error#PEBKAC/PEBCAK) as usual +_+
